src/Rings/FreeAssAlgIdeal.jl

```diff
@@ -45,17 +35,17 @@ end
 
 Return the two-sided ideal of $R$ generated by $g$.
 """
-function ideal(R::FreeAssAlgebra, g::Vector{T}) where T <: FreeAssAlgElem
+function ideal(R::FreeAssAlgebra, g::Vector{<:T}) where T <: FreeAssAlgElem
```

Comment:
you shouldn't need the T here
```suggestion
function ideal(R::FreeAssAlgebra, g::Vector{FreeAssAlgElem}) 
```